docs/api/supervisor/endpoints.md

```diff
@@ -1434,49 +1440,77 @@ Get network information.
 
 </ApiEndpoint>
 
-<ApiEndpoint path="/network/<interface>/info" method="get">
-
+<ApiEndpoint path="/network/interface/<interface>/info" method="get">
```

Comment:
```suggestion
<ApiEndpoint path="/network/interface/<interface>/info" method="get">

```

The space is needed since we use markdown on the next line